frontend/src/types/Theme.ts

```diff
@@ -2,6 +2,7 @@ export interface Header {
     nextExperimentButtonText: string;
     aboutButtonText: string;
     showScore: boolean;
+    totalScore: BigInteger;
```

Comment:
Again, I don't think the total score should be part of a `Theme` interface.

Reply:
changed to header